cfme/infrastructure/config_management.py

```diff
@@ -544,27 +545,32 @@ def quad_name(self):
             return '{} Configuration Manager'.format(self.name)
 
 
-def get_config_manager_from_config(cfg_mgr_key):
+def get_config_manager_from_config(cfg_mgr_key, appliance=None, mgr_type=None):
```

Comment:
As per `load_from_yaml` classmethod. I think appliance should required arg.

Reply:
When this is used in the test phase, we don't always have an appliance, and we can instantiate a config manager object without one.

This will be updated/removed in @john-dupuy work to make the configmanager a "real" provider class.